util/lang_def/restest_lex.py

```diff
@@ -41,17 +42,21 @@ class Lexer:
         "COLON",
         "DOT",
         "COMMA",
+        "COMMENT",
+        "HEADER_PARAMETERS",
     ] + list(reserved.values())
 
     # Macros
 
     letter = r"[a-zA-Z]"
     identifier = letter + r"(\d|" + letter + r"|_|-)*"
+    header_params = r"([\w]+:[\w]+,?[\s]*)+"
 
     # REGEX
     t_ignore = " \t"
+    t_ignore_COMMENT = r"(\#\*[\s\S]*?\*\#)|(\#.*)"
 
-    t_OPERATOR = r"(==|!=|>=|<=|>|<)"
+    t_OPERATOR = r"(==|!=|>|<|>=|<=)"
```

Comment:
In this line the <= and >= was before the < and > to prevent lexer not recognizing = symbol alone

Reply:
I think you have fixed this before, not sure why it ended up like this. I'll make sure to correct it.